openedx/core/djangoapps/theming/template_loaders.py

```diff
@@ -27,21 +27,21 @@ class ThemeFilesystemLoader(FilesystemLoader):
     is_usable = True
     _accepts_engine_in_init = True
 
-    def get_template_sources(self, template_name, template_dirs=None):
+    def __init__(self, engine, dirs=None):
+        if not dirs:
+            self.dirs = engine.dirs
+        theme_dirs = self.get_theme_template_sources()
+        if isinstance(theme_dirs, list):
+            self.dirs = theme_dirs + self.dirs
+        super(ThemeFilesystemLoader, self).__init__(engine, self.dirs)
+
+    def get_template_sources(self, template_name):
```

Comment:
Do we even need this anymore?  All it does now is make a list out of the superclass's implementation.

Reply:
right, we can just remove it.